bigbluebutton-html5/imports/ui/components/audio/container.jsx

```diff
@@ -132,7 +133,7 @@ export default withModalMounter(injectIntl(withTracker(({ mountModal, intl }) =>
       Service.init(messages, intl);
       Service.changeOutputDevice(document.querySelector('#remote-media').sinkId);
       if (!autoJoin || didMountAutoJoin) return;
-
+      Session.set('audioModalIsOpen', true);
```

Comment:
Why not place this line in `openAudioModal`? Here it looks like it will be set to true while we still have the webcam selector (if also set to autojoin)